Run Jörmungandr integration tests on genesis/praos in CI

Anviking opened the pull request:
# Issue Number

#698 

# Overview

<!-- Detail in a few bullet points the work accomplished in this PR -->

- [x] I have migrated the "minimum setup" for praos/genesis to our previously bft-config.
- [x] I have set `consensus_genesis_praos_active_slot_coeff` to `1`. (It seems to make the stake pool produce blocks every slot)
 


# Comments

<!-- Additional comments or screenshots to attach if any -->

<!-- 
Don't forget to:

 ✓ Self-review your changes to make sure nothing unexpected slipped through
 ✓ Assign yourself to the PR
 ✓ Assign one or several reviewer(s)
 ✓ Once created, link this PR to its corresponding ticket
 ✓ Acknowledge any changes required to the Wiki
-->

Anviking committed:
Run integration tests on genesis/praos

KtorZ commented on lib/jormungandr/test/data/jormungandr/genesis.yaml:
```diff
@@ -3062,3 +3062,14 @@ initial:
+      - address: ca1s5nwh0ur9qj4tg5hx334ht9zvqwn9ddz8rv5acwdl997zmqpswdujuwqu9j
+        value: 100000000000000
+      - address: ca1s5498yesws3eee80nxlxqlqetpdstam9x0hkcs7f2k4386uyulydvmzrtja
+        value: 100000000000000
```
Do we need this :thinking: ? Aren't the initial certs sufficient?

Anviking replied:
Our existing addresses are single-addresses which shouldn't be possible to stake with.

I simplified this PR to only add a single account-address with `1` lovelace that delegates to the pool.

I guess we would like to test that we can stake faucet-ada from the wallet in the future, and that we would like to make them into grouped addresses.

piotr-iohk commented on lib/jormungandr/test/data/jormungandr/genesis.yaml:
```diff
@@ -3062,3 +3062,14 @@ initial:
     - address: ca1swl53wlqt5dnl63e0gnf8vpazgt6g5mq384dmz72329eh4m8z7e5un8q6lg
       value: 100000000000
+
+  # Funds & certificates for the single stake pool
```
For the sake of testing listing of stake pools maybe it would be worth adding a few stake pools, with different initial stakes. Say 3 stake pools (high stake, moderate stake and very low stake). 

Ofc, this can be done later...

Anviking replied:
Good point. I think we can delay it to the ticket where it will first be useful.

Anviking committed:
Rename --bft-leaders cli arg to --secret

Since it also should work on genesis/praos. This is also the term jörmungandr uses.

KtorZ reviewed (approved):
LGTM :+1:
